hokusai/production.yml

```diff
@@ -121,6 +121,23 @@ spec:
               serviceName: kaws-web-internal
               servicePort: http
 
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: kaws-internal
```

Comment:
Conventionally we'd just call this `kaws`, but that's occupied by the public ingress defined above. In the follow-up PR dropping that, I'll probably update this name.